### PR TITLE
FIX #33 - Left indentation mathematical equations 

### LINKED ISSUE #33: Remove the left-hand labels and left-indent equations in the mathematical models
I would like the ODEs rendered without this left border text

image

Let's **left indent all the equations**. 

For example, an SIR model without vital dynamics under the pseudo mass action formulation would look like this

image

Another example, an SEIRD model under the true mass action formulation would like this

image

_Originally posted by @ashokkrish in https://github.com/ashokkrish/episim/issues/31#issuecomment-2106118570_

### PR DESCRIPTION
**Pull request by kle6951**

### CODE REVIEW

**Comment by kle6951** [comment]
I have done the code cleaning and completed the requested change. There will be some minor tasks updating this part for PMA and TMA equations format. Please let me know if it needs any further improvement and correctness.

File: R/mathemtical_equations.R
Comment (bryce-carson) [inline R/mathemtical_equations.R:1]: The mathematical_equations file looks good to me!

File: R/model_LaTeX.R [hunk 1]
```diff
@@ -1,113 +1,67 @@
...
+  '    justify-content: flex-start;',
+  '  }',
+  '  .equation-block {',
+  '    margin: 0;',
+  '    padding: 0;',
+  '  }',
+  '</style>'
+))
+
+# Helper function to generate the LaTeX content
```
Comment (bryce-carson) [inline R/model_LaTeX.R:15]: This is a good use of a function. :) This is how we **R**oll. 😎

**Review by bryce-carson — APPROVED** [review]
Looks good to me!

**Pull request merged**